composer.json

```diff
@@ -21,6 +21,11 @@
         ]
     },
     "extra": {
-        "altis": {}
+        "altis": {
+            "install-overrides": [
+                "altis/consent",
```

Comment:
`altis/consent` is a package, doesn't need to be in here

Reply:
We need to make `altis/consent` into a `wordpress-plugin` package type really, I added an upstream issue to do that. It's copied from the list in the core installer right now so I think it's safer to include it here anyway.